tests/e2e/vmgroups_test.go

```diff
@@ -431,3 +432,138 @@ func (vg *VmGroupTest) TestVmGroupVolumeClone(c *C) {
 
 	misc.LogTestEnd(c.TestName())
 }
+
+// Restore VMgroup
+func (vg *VmGroupTest) RestoreVmgroup(c *C) {
```

Comment:
Nit: This function should start with small letter since we don't really want to export it.